test/test_integration_credentials.py

```diff
@@ -211,30 +217,32 @@ def test_https_netrc_is_created_with_host(self):
         output = self.invoke_cli(self.cli_auth_params,
                                  ['login', '-i', 'host/somehost', '-p', f'{extract_api_key_from_message}'])
 
-        self.validate_netrc(f"{self.client_params.hostname}", "host/somehost", extract_api_key_from_message)
+        self.validate_credentials(f"{self.client_params.hostname}", "host/somehost", extract_api_key_from_message)
         self.assertIn("Successfully logged in to Conjur", output.strip())
 
+    @integration_test()
+    def test_working_with_keystore(self):
+        cred_store,_ = CredentialStoreFactory().create_credential_store()
+        self.assertEquals(type(cred_store) , type(KeystoreCredentialsProvider()))
+
     '''
-    Validates when a user can logout successfully
-    '''
+        Validates logout doesn't remove another entry not associated with Cyberark
```

Comment:
fix indentation here to be aligned with '''

Reply:
not just about not being associated with Cyberark/Conjur. It is just that we don't delete an entry that is irrelevant to what is required

So something like:
Validates logout doesn't remove another irrelevant entry